### PR TITLE
Fix iOS takeSnapshot not working

### PR DESCRIPTION
**Pull request by Antonimo**
Fix iOS takeSnapshot not working

### CODE REVIEW

File: ios/Classes/SwiftFlutterVlcPlayerPlugin.swift [hunk 1]
```diff
@@ -147,6 +147,8 @@ public class VLCView: NSObject, FlutterPlatformView {
                     result(FlutterMethodNotImplemented)
                     return
                 }
+            } else {
+                print("no arguments")
```
Comment (solid-software) [inline ios/Classes/SwiftFlutterVlcPlayerPlugin.swift:151]: We usually try to avoid print statements.

File: lib/flutter_vlc_player.dart [hunk 2]
```diff
@@ -320,7 +320,7 @@ class VlcPlayerController {
   }
 
   Future<Uint8List> takeSnapshot() async {
-    var result = await _methodChannel.invokeMethod("getSnapshot");
+    var result = await _methodChannel.invokeMethod("getSnapshot", {'test': ''});
```
Comment (solid-software) [inline lib/flutter_vlc_player.dart:323]: Do we need to pass this 'test' data always or it can be removed?

Comment (Antonimo) [reply lib/flutter_vlc_player.dart:323]: it was a quick fix, if no arguments then the invoke is ignored

Comment (mitchross) [reply lib/flutter_vlc_player.dart:323]: just change 'test' to 'getSnapShot' then

File: ios/Classes/SwiftFlutterVlcPlayerPlugin.swift (hunk 1 above)
Comment (mitchross) [reply ios/Classes/SwiftFlutterVlcPlayerPlugin.swift:151]: Debug Log statement may be better

File: lib/flutter_vlc_player.dart (hunk 2 above)
Comment (mitchross) [reply lib/flutter_vlc_player.dart:323]: @Antonimo can you make the change?

**Comment by Antonimo** [comment]
@solid-software @mitchross updated

**Comment by mitchross** [comment]
@Antonimo  did you push up your changes? I dont see it.

**Comment by solid-software** [comment]
Thanks @Antonimo @mitchross!

**Pull request merged**